eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/task/MessageTransferTask.java

```diff
@@ -181,25 +181,20 @@ public void run() {
 
     private CloudEvent addTimestamp(CloudEvent event, Command cmd, long sendTime) {
         if (cmd == RESPONSE_TO_SERVER) {
-            event = CloudEventBuilder.from(event)
-                .withExtension(EventMeshConstants.RSP_C2EVENTMESH_TIMESTAMP,
-                    String.valueOf(startTime))
-                .withExtension(EventMeshConstants.RSP_EVENTMESH2MQ_TIMESTAMP,
-                    String.valueOf(sendTime))
-                .withExtension(EventMeshConstants.RSP_SEND_EVENTMESH_IP,
-                    eventMeshTCPServer.getEventMeshTCPConfiguration().getEventMeshServerIp())
-                .build();
+            return getCloudEvent(event, sendTime, EventMeshConstants.RSP_C2EVENTMESH_TIMESTAMP, EventMeshConstants.RSP_EVENTMESH2MQ_TIMESTAMP,
+                EventMeshConstants.RSP_SEND_EVENTMESH_IP);
         } else {
-            event = CloudEventBuilder.from(event)
-                .withExtension(EventMeshConstants.REQ_C2EVENTMESH_TIMESTAMP,
-                    String.valueOf(startTime))
-                .withExtension(EventMeshConstants.REQ_EVENTMESH2MQ_TIMESTAMP,
-                    String.valueOf(sendTime))
-                .withExtension(EventMeshConstants.REQ_SEND_EVENTMESH_IP,
-                    eventMeshTCPServer.getEventMeshTCPConfiguration().getEventMeshServerIp())
-                .build();
+            return getCloudEvent(event, sendTime, EventMeshConstants.REQ_C2EVENTMESH_TIMESTAMP, EventMeshConstants.REQ_EVENTMESH2MQ_TIMESTAMP,
+                EventMeshConstants.REQ_SEND_EVENTMESH_IP);
```

Comment:
You may give these parameters newlines to improve readability, or create a new local variable to pass the parameter. I would suggest the former.

Reply:
Do you mean like this?

```java
private CloudEvent addTimestamp(CloudEvent event, Command cmd, long sendTime) {
      if (cmd == RESPONSE_TO_SERVER) {
          return getCloudEvent(event, sendTime,
              EventMeshConstants.RSP_C2EVENTMESH_TIMESTAMP,
              EventMeshConstants.RSP_EVENTMESH2MQ_TIMESTAMP,
              EventMeshConstants.RSP_SEND_EVENTMESH_IP);
      } else {
          return getCloudEvent(event, sendTime,
              EventMeshConstants.REQ_C2EVENTMESH_TIMESTAMP,
              EventMeshConstants.REQ_EVENTMESH2MQ_TIMESTAMP,
              EventMeshConstants.REQ_SEND_EVENTMESH_IP);
      }
  }
```

Reply:
It seems better. You might want to ask @lrhkobe's opinion.

Reply:
ping @lrhkobe

Reply:
I prefer to keep the previous way.
Because the readability of this method `private CloudEvent getCloudEvent(CloudEvent event, long sendTime, String times, String mq, String ip)`  is not good.

Such as:
```
 CloudEventBuilder.from(event)
            .withExtension(times, String.valueOf(startTime))
            .withExtension(mq, String.valueOf(sendTime))
```
The parameter ( `times` and `mq` ) and the value passed in  seems not match.

Reply:
So. should I keep it the same as before? Since I just did it as the issue told.